libnavigation-ui/src/main/java/com/mapbox/navigation/ui/map/building/BuildingLayerSupport.kt

```diff
@@ -34,14 +30,11 @@ internal class BuildingLayerSupport {
      * expression and which building is closest to the coordinate.
      * @return an [Expression.all] expression
      */
-    fun getBuildingFilterExpression(queryLatLng: LatLng, buildingInt: Int): Expression {
+    fun getBuildingFilterExpression(buildingString: String): Expression {
         return all(
                 eq(get("extrude"), "true"),
-                eq(get("type"), "building"),
```

Comment:
For anyone wondering, `eq(get("type"), "building"),` was removed in this pr because `Feature`s in the `building` layer can have all sorts of various types. https://wiki.openstreetmap.org/wiki/Key:building . We want the footprint/extrusion to be highlighted no matter the type (`building`, `school`, `residential`, etc.)